nose_parameterized/parameterized.py

```diff
@@ -211,11 +211,53 @@ def expand(cls, input, testcase_func_name=None):
             >>>
             """
 
-        def parameterized_expand_wrapper(f):
+        def doc_expansion(f, args, kwargs, is_method):
+            offset = 1 if is_method else 0
+            vns = inspect.getargspec(f)[0][offset:]
```

Comment:
We can assume 2.6+ so we can use the namedtuple that `getargspec(…)` returns: `argspec = inspect.getargspec(f); args = argspec.args`